src/TTImp/TTImp.idr

```diff
@@ -728,7 +728,7 @@ implicitsAs n defs ns tm
         -- corresponding to the variables bound in the parent function.
         -- So we first peel off all of the explicit quantifiers corresponding
         -- to these variables.
```

Comment:
Shouldn't this comment be changed?

Reply:
Thanks. Yes, it should.  I forgot to go back and take care of this.  I've updated it to mention that parameter blocks also contribute variables that may not be explicit.